src/util/math.h

```diff
@@ -38,40 +45,38 @@ inline bool even(T value) {
 #pragma intrinsic(fabs)
 #endif
 
-inline int roundUpToPowerOf2(int v) {
-    int power = 1;
+constexpr unsigned int roundUpToPowerOf2(int v) {
+    DEBUG_ASSERT(v >= 0);
+#ifdef __cpp_lib_bitops
+    const auto uv = static_cast<unsigned int>(v);
```

Comment:
This should be moved in the common part and the legacy workaround should use uv as well